docs/src/main/paradox/openshift.md

```diff
@@ -0,0 +1,99 @@
+# Deploy on OpenShift
+
+### Prerequisites
+
+Install the following:
+
+* [Docker](https://docs.docker.com/install/)
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* OpenShift's CLI: [`oc`](https://docs.openshift.com/container-platform/3.10/cli_reference/get_started_cli.html#installing-the-cli)
+* [Sbt](https://www.scala-sbt.org/)
+
+
+#### Preface
+
+There are [multiple flavors](https://www.openshift.com/products?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) of `oc` and OpenShift. This guide was tested with:
+
+```
+$ oc version
+
+oc v3.10.45
+kubernetes v1.10.0+b81c8f8
+features: Basic-Auth
+
+Server https://centralpark.lightbend.com:443
+openshift v3.10.45
+kubernetes v1.10.0+b81c8f8
+```
+
+Where `centralpark.lightbend.com` is an [OpenShift Container Platform](https://www.openshift.com/products/container-platform/).
+
+### Running
+
+Login to OpenShift from your terminal and create the OpenShift project:
+
+```
+## obtain the token from https://<your-openshift-server>/console/command-line
+export TOKEN=<my-token>
+export OPENSHIFT_PROJECT=play-scala-grpc-example
+export IMAGE=play-scala-grpc-example
+export TAG=1.0-SNAPSHOT
+
+oc login https://centralpark.lightbend.com --token=$TOKEN
+oc new-project $OPENSHIFT_PROJECT
+```
+
+Create the docker image of your application and push it to the image registry (for this example we're using the Red Hat Container Registry in CentralPark).
+
+```bash
+sbt docker:publishLocal
+
+export DOCKER_REGISTRY_SERVER=docker-registry-default.centralpark.lightbend.com
+export DOCKER_REGISTRY=$DOCKER_REGISTRY_SERVER/$OPENSHIFT_PROJECT
+
+docker login -p $TOKEN -u unused $DOCKER_REGISTRY_SERVER
+docker tag $IMAGE:$TAG $DOCKER_REGISTRY/$IMAGE:$TAG
+docker push $DOCKER_REGISTRY/$IMAGE:$TAG
+```
```

Comment:
This script publishes the image locally and then pushes it to centralpark. I'd like to publish it directly into centralpark.

Note: I followed the instruction in https://docs.google.com/document/d/1LmNoH3k37OjF98dkrhIXyuJtTOc-GiPduHfujJsiTf0/edit#